src/javascript/app/pages/user/account/settings/iphistory/iphistory.ui.js

```diff
@@ -13,12 +13,13 @@ const IPHistoryUI = (() => {
         const browser      = data.browser;
         let browser_string = browser ? `${browser.name} v${browser.version}` : 'Unknown';
         if (browser && browser.app) {
-            browser_string += ` ${browser.app} app`;
+            browser_string = ` ${browser.name} ${browser.app} v/${browser.version}`;
```

Comment:
Making browser data consistent with deriv app. In deriv app - we display browser data like - **Android Deriv GO v/0.8.8+18**